tasks/tuning.yml

```diff
@@ -19,6 +19,20 @@
     when: host_preparation_is_tuning and host_preparation_sysctl_vars is defined
     register: host_preparation_configure_sysctl
 
+  - name: Checking rc.local
+    stat:
+      path: /etc/rc.local
+    register: rc
+  
+  - name: Create new rc.local
+    template:
+      src: rc.local.j2
+      dest: /etc/rc.local
+      owner: root
+      group: root
+      mode: 0755
```

Comment:
Use parameter "force: no" so it will going to run only when file does not exist and you can remove Checking rc.local task
https://docs.ansible.com/ansible/latest/modules/template_module.html

Reply:
fixed on 130aee1

Reply:
fixed on 130aee1